### PR TITLE
New Cost Estimation pages for TFC

### PR DESCRIPTION
**Pull request by thrashr888**
Cost Estimation is given its own section in the docs. Sub-pages list our supported resources.

image

image

### CODE REVIEW

File: content/source/docs/cloud/cost-estimation/aws.html.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+layout: "cloud"
+page_title: "AWS - Cost Estimation - Terraform Cloud"
+---
+
+# Supported AWS resources in Terraform Cloud Cost Estimation
+
+Cost Estimation support for AWS allows you to expose and control many terraform resources within AWS. Setup for AWS credenitals in Cost Estimation in Terraform Enterprise is found in the [Application Administration docs](/docs/enterprise/admin/integration.html).
+
+Resources that incurr costs are marked as `IC`. Resources marked `DNI` do not have costs associated with them.
```
Comment (glenngillen) [inline content/source/docs/cloud/cost-estimation/aws.html.md]: "incur"?

File: content/source/docs/cloud/cost-estimation/azure.html.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+layout: "cloud"
+page_title: "Azure - Cost Estimation - Terraform Cloud"
+---
+
+# Supported Azure resources in Terraform Cloud Cost Estimation
+
+Cost Estimation support for Azure allows you to expose and control many terraform resources within Azure. Setup for Azure credenitals in Cost Estimation in Terraform Enterprise is found in the [Application Administration docs](/docs/enterprise/admin/integration.html).
+
+Resources that incurr costs are marked as `IC`. Resources marked `DNI` do not have costs associated with them.
```
Comment (glenngillen) [inline content/source/docs/cloud/cost-estimation/azure.html.md]: same.

File: content/source/docs/cloud/cost-estimation/gcp.html.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: "cloud"
+page_title: "GCP - Cost Estimation - Terraform Cloud"
+---
+
+# Supported GCP resources in Terraform Cloud Cost Estimation
+
+Cost Estimation support for GCP allows you to expose and control many terraform resources within GCP. Setup for GCP credenitals in Cost Estimation in Terraform Enterprise is found in the [Application Administration docs](/docs/enterprise/admin/integration.html).
+
+Resources that incurr costs are marked as `IC`. Resources marked `DNI` do not have costs associated with them.
```
Comment (glenngillen) [inline content/source/docs/cloud/cost-estimation/gcp.html.md]: same.

**Comment by thrashr888** [comment]
The latest changes

image

**Comment by thrashr888** [comment]
For the "Stuff that might cost you money indirectly, possibly via usage metrics that we can't predict." case, we'll leave those as unsupported for now and thus not in this list. We wouldn't add them as DNI so I think we're safe here.

Support for usage-based pricing might come along with future custom pricing work.

**Comment by thrashr888** [comment]
i'm going to hold off on merging this until cost-estimation#139 is merged, which applies the DNI list. otherwise, the list here is a bit premature. thanks for helping prep it!

**Comment by nfagerlund** [comment]
Ok, great! (I must have misinterpreted something. 😅) In that case, what do you think about "incurs cost" vs. (blank), with a note up top saying that many supported resources don't incur costs?

> Below is a list of resources that cost estimation supports so far. Resources that incur costs are marked as such; other resources do not have costs associated with them.
> 
> | Resource | Status |
> |-|-|
> | aws_instance | incurs cost |
> | aws_lb | incurs cost |
> | aws_rds_cluster | incurs cost |
> | aws_acm_certificate_validation |  |
> | aws_alb_listener |  |
> | aws_alb_listener_rule |  |
> | aws_alb_target_group |  |

**Comment by thrashr888** [comment]
Works for me! I just merged the changes in cost-estimation so we're good to go.

**Pull request merged**